Makefile

```diff
@@ -57,15 +57,18 @@ test-with-cov :
 
 .PHONY : install
 install :
+	# Making sure the typing backport isn't installed.
+	pip uninstall -y typing
```

Comment:
Note: this will have exit code 0 even if typing is not installed

Reply:
Yes, but it prints a warning. That's why I did the `grep` thing.